<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BusLines</class>
 <widget class="QDialog" name="BusLines">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Otobüs Hat Oluşturma</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Ekle</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>50</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Duraklar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>50</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Hat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>420</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Kaydet</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>45</y>
     <width>521</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Durak kısmından ilgili durağın üzerine tıklayarak &quot;Ekle&quot; butonu vasıtasıyla hattınıza durak ekleyebilirsiniz.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Hattınızın numarasını giriniz. (Örneğin; BUS-LINES-001)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>210</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>210</width>
     <height>600</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>210</width>
     <height>600</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
